testing/tests/DevExpress.ui.widgets.dataGrid/columnChooser.integration.tests.js

```diff
@@ -83,6 +83,82 @@ QUnit.module('Column chooser', baseModuleConfig, () => {
         assert.ok($overlayWrapper.hasClass('dx-datagrid-column-chooser-mode-select'), 'has select mode class');
     });
 
+    QUnit.test('Correct runtime changing of sortOrder (string)', function(assert) {
```

Comment:
It is not necessary to write tests with real dom elements for this functionality. It will be better to write tests for getChooserColumns method only in columnController tests.